tests/playwright/src/model/pages/cli-tools-page.ts

```diff
@@ -97,11 +100,109 @@ export class CLIToolsPage extends SettingsPage {
         console.log(`Dialog for tool ${toolName} was not visible. Proceeding.`);
       }
 
+      await playExpect
+        .poll(async () => await this.getCurrentToolVersion(toolName), { timeout: timeout })
+        .toContain(version);
```

Comment:
If  the version parameter is empty we should not check if version contains `''`.